frontend/src/container/Login.jsx

```diff
@@ -157,12 +157,14 @@ class Login extends React.Component {
 
     const { name, login, password } = event.target
 
-    if (name.value === '' || login.value === '' || password.value === '') {
+    const email = login.value.trim()
```

Comment:
I would leave the variable name as "login" because in this case it can be an email or a username.